### DISCUSSION TITLE
Thoughts/Ideas based on Published File Review

### DISCUSSION THREAD

**Discussion by sbenphia** [CMSgov/price-transparency-guide#526] [Ideas]
Has anyone started downloading the published files and reviewed them in order to identify best practices, interesting strategies, and issues that should be remedied in future design and direction?  

To start - I have noticed some interesting disclaimers and statements such as: "UHC disclaims the fitness of the data herein for any other purpose, including use in any third-party database or product. Any such use and any liability attached thereto is at the risk of such third party."  versus: "The machine readable files are formatted to allow researchers, regulators, and application developers to more easily access and analyze data."

Some of these statements seem a little problematic, such as a disclaimer identifying the file isn't fit for analysis on one hand (UHC) and intended for analysis on another hand (CIGNA).  Just an observation I find interesting.  Also - UHC has over 79,000 links to files, while Cigna has one, definitely two approaches.

**Comment by jeffld** [comment]
The lawyers are probably scrambling to use whatever means they can to protect the company.  

Since you mention CIGNA, I know for a fact that Cigna does not publicly post all the MRF data.  They require a proprietary method which is behind authentication in order to access the files.
